test/pythonml/test_pre_processing_flavors.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+import importlib
+import os
+import shutil
+import unittest
+from parameterized import parameterized
+import pandas
+import numpy as np
+from fixtures.unittest_baseclass import BaseUnitTest
+
+
+def load_test_train_targets_and_descriptors():
+    """
+    Loads a set of commonly used pickle files from an updated location in the context object
+    """
+    # import and roload settings to update the context object's path dir
+    import settings
+    importlib.reload(settings)
+    train_target = settings.context.load("train_target")
+    train_descriptors = settings.context.load("train_descriptors")
+    test_target = settings.context.load("test_target")
+    test_descriptors = settings.context.load("test_descriptors")
+    return train_target, train_descriptors, test_target, test_descriptors
+
+
+class TestPreProcessingScalerFlavors(BaseUnitTest):
+    """
+    This class performs unittests for the pre_processing scaler flavors.
+    We may only check regression data here, and assume it would work for the other data categories.
```

Comment:
We should also do this for the other categories, since these units do behave differently depending on the type of problem. Not necessarily as part of this code review, but I'd make a task in Jira to finish this up for clustering / classification

Reply:
I'll make this a separate Jira task, to test Classification and Clustering scenarios. The scaler may have different behavior depending on problem category